lib/grape/validations/validators/coerce.rb

```diff
@@ -54,7 +54,14 @@ def coerce_value(type, val)
         return val || Set.new if type == Set
         return val || {}      if type == Hash
 
-        converter = Virtus::Attribute.build(type)
+        # To support custom types that Virtus can't easily coerce, pass in an
+        # explicit coercer. Custom types must implement a `parse` class method.
+        converter_options = {}
+        if ![DateTime, Date, Time].include?(type) && type.respond_to?(:parse)
```

Comment:
I think we want a coercer if this is not a known Virtus type or something like that, no? It feels wrong that we're checking for a list of known types, or at least that responsibility should be elsewhere (in Virtus or an extension of Virtus). I think the logic should be something like this:

```ruby
converter_options = {}
case type
when Virtus::Attribute.something_that_can_be_coerced_or_virtus_knows_about_it?
   # do nothing
when type.respond_to?(:parse)
   convert_options[:coercer] = ...
end
```

What do you think?